tests/executors/test_executor_loader.py

```diff
@@ -68,7 +68,13 @@ def test_no_executor_configured(self):
     )
     def test_should_support_executor_from_core(self, executor_name):
         with conf_vars({("core", "executor"): executor_name}):
-            executor = executor_loader.ExecutorLoader.get_default_executor()
+            # These executors are deprecated and will be removed in Airflow 3.0 but we still need to support
+            # them in Airflow 2.10.X
+            if executor_name == "CeleryKubernetesExecutor":
+                with pytest.warns(RemovedInAirflow3Warning):
+                    executor = executor_loader.ExecutorLoader.get_default_executor()
```

Comment:
LocalK8s also?

Reply:
If you look at the test parameterization above, LocalKubernetesExecutor is not part of this test, so no need :)